includes/REST/StoreController.php

```diff
@@ -52,6 +52,99 @@ public function register_routes() {
                     'methods'             => WP_REST_Server::CREATABLE,
                     'callback'            => [ $this, 'create_store' ],
                     'permission_callback' => [ $this, 'permission_check_for_manageable_part' ],
+                    'args'                => [
+                        'user_login' => [
+                            'required'    => false,
+                            'type'        => 'string',
+                            'description' => __( 'The username for the store owner. If not provided, it will be auto-generated.', 'dokan-lite' ),
+                        ],
+                        'email' => [
+                            'required'    => true,
+                            'type'        => 'string',
+                            'format'      => 'email',
+                            'description' => __( 'The email address for the store owner.', 'dokan-lite' ),
+                        ],
+                        'store_name' => [
+                            'required'    => true,
+                            'type'        => 'string',
+                            'description' => __( 'The name of the store.', 'dokan-lite' ),
+                        ],
+                        'social' => [
+                            'required'    => false,
+                            'type'        => 'array',
+                            'description' => __( 'An array of social media details for the store.', 'dokan-lite' ),
+                        ],
+                        'payment' => [
+                            'required'    => false,
+                            'type'        => 'array',
```

Comment:
Pls define the `array` type properly to  enforce the format of each item in the array. Pls see https://developer.wordpress.org/rest-api/extending-the-rest-api/schema/#arrays